src/data/content/design-system/come-iniziare.yaml

```diff
@@ -4,6 +4,9 @@
 metadata:
   template:
     name: design-system-index
+  redirect_from:
+    - /design-system-del-paese
```

Comment:
@gianvitofanelli the "home" of the design system is the design system page, not this "Come iniziare" second level. Is it not? 🤔

Reply:
You're right, let's ditch it